docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
   migration:
     build: .
     command: bash -c "sh wait-for-postgres.sh && python manage.py migrate && python manage.py loaddata ops_site/fixtures/fake_data.json"
+    volumes:
+      - ./opre_ops:/${CI}opre_project
```

Comment:
Similar to the `volumes` property in the other Docker service. This one allows the `migration` container to run the most recent migrations without having to rebuild it. You can instead run `docker compose up -d` at any time to apply the latest migrations.